docs/docs/components/reanimated-drawer-layout.mdx

```diff
@@ -90,7 +93,11 @@ If not enabled, the gesture will require click + drag, with `enableTrackpadTwoFi
 ### `children`
 
 either a component that's rendered in the content view or a function.
-When provided as a function, it accepts a `progress` SharedValue parameter that indicates the animation progress, similar to the [`renderNavigationView`](#rendernavigationview) function.
+When provided as a function, it is provided with a `progress` parameter.
```

Comment:
**_Provided_** twice in the same, short sentence doesn't sound good 😅

Reply:
does `supplied` work for the second `provided`?

Reply:
What about simply:

```
If `children` is a function, it is provided with a `progress` parameter.
```

Reply:
Sounds good, fixed in [aa73f47](https://github.com/software-mansion/react-native-gesture-handler/pull/3188/commits/aa73f4741ba03ae51b59e5a30a3cff589acb6212)